scripts/build_bus_regions.py

```diff
@@ -77,6 +78,8 @@ def custom_voronoi_partition_pts(points, outline, add_bounds_shape=True, multipl
     from scipy.spatial import Voronoi
     from shapely.geometry import Point, Polygon
 
+    outline = unary_union(outline)
```

Comment:
By looking at the docstring there is a refuse in the docstring with the option no_multipolygons that is not used.
In pypsa-eur there have been some changes.

What are the implications of keeping the multipolygon here?

Reply:
Well, it looks like our docstring is simply outdated (see commit https://github.com/PyPSA/pypsa-eur/commit/97fbf77ff8ddb7da8bb75602b73853e319854cb5 in PyPSA-Europe). Fixed